app/src/main/java/org/wikipedia/activity/BaseActivity.java

```diff
@@ -88,6 +95,14 @@
             getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         }
 
+        PeriodicWorkRequest editorReactivationWorker =
+                new PeriodicWorkRequest.Builder(SuggestedEditsReactivationWorker.class, 1, TimeUnit.DAYS)
+                        .setConstraints(new Constraints.Builder().build())
+                        .setInitialDelay(1, TimeUnit.HOURS)
+                        .build();
+        WorkManager.getInstance(this)
+                .enqueueUniquePeriodicWork("suggested-edits-worker", KEEP, editorReactivationWorker);
```

Comment:
Not sure why, but I never got the WorkManager to actually trigger the doWork callback, even when I modified the TimeUnits to be `SECONDS`...

Is there a reason we can't just reuse our existing `NotificationPollBroadcastReceiver`, which already runs periodically in the background?

Reply:
Looks like the `PeriodicWorkRequest` requires at least 15 minutes, and that's why it did not trigger the task.
https://medium.com/androiddevelopers/workmanager-periodicity-ff35185ff006